File: data/EXP01/python/2300/2300.output_1.py
# @lc app=leetcode id=2300 lang=python3
from typing import List

class Solution:
    def successfulPairs(self, spells: List[int], potions: List[int], success: int) -> List[int]:
        pairs = []
        for spell in spells:
            count = 0
            for potion in potions:
                if spell * potion >= success:
                    count += 1
            pairs.append(count)
        return pairs

File: data/EXP01/golang_no_system_var/2300/2300.output_1.go
func successfulPairs(spells []int, potions []int, success int64) []int {
    res := make([]int, len(spells))

    for i := 0; i < len(spells); i++ {
        for j := 0; j < len(potions); j++ {
            if int64(spells[i])*int64(potions[j]) >= success {
                res[i]++
            }
        }
    }

    return res
}


    potions := []int{1, 2, 3, 4, 5}
    success := int64(7)

    fmt.Println(successfulPairs(spells, potions, success)) // Output: [4, 0, 3]
}
